lib/spack/spack/ci/__init__.py

```diff
@@ -879,7 +879,7 @@ def reproduce_ci_job(url, work_dir, autostart, gpg_url, runtime, use_local_head)
     """
     work_dir = os.path.realpath(work_dir)
     if os.path.exists(work_dir) and os.listdir(work_dir):
-        raise SpackError(f"Cannot run reproducer in non-emptry working dir:\n  {work_dir}")
+        raise SpackError(f"Cannot run reproducer in non-empty working dir:\n  {work_dir}")
```

Comment:
Can I fix a typo in an error message? Does a change here break something?

Reply:
that's fine